client/deis.py

```diff
@@ -800,15 +804,20 @@ def auth_login(self, args):
         password = args.get('--password')
         if not password:
             password = getpass('password: ')
+        disable_ssl_verify = args.get('--disable_ssl_verify')
```

Comment:
We seem to prefer using hyphens rather than underscores for option names, so this would become `--disable-ssl-verify`. I'm also wondering if `--no-ssl-verify` works just as well - it's certainly shorter!